build-tools/mono-runtimes/ProfileAssemblies.projitems

```diff
@@ -260,8 +260,6 @@
     <MonoTestAssembly Include="monodroid_System.Xml.Linq_test.dll">
       <SourcePath>System.Xml.Linq</SourcePath>
     </MonoTestAssembly>
-    <MonoTestAssembly Include="nunitlite.dll">
-      <SourcePath>lib/monodroid</SourcePath>
-    </MonoTestAssembly>
+    <MonoTestRunner Include="nunitlite.dll" />
```

Comment:
Is this change correct & appropriate? There doesn't appear to be *any* usage or reference to `MonoTestRunner` in my current checkout. I assume this is a mistake.